internal/split/split_service.go

```diff
@@ -0,0 +1,88 @@
+package split
+
+import (
+	"io/ioutil"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/splitio/go-client/v6/splitio/client"
+	"github.com/splitio/go-client/v6/splitio/conf"
+
+	configAPI "github.com/newrelic/newrelic-cli/internal/config/api"
+)
+
+// Split.io client API keys
+// These keys are client-facing and are only used to fetch splits.
+// There is no security risk in exposing these keys, as the only purpose they
+// serve is to retrieve experiments and can not be used to modify anything
+// within our internal Split.io system.
+var (
+	prod      = "8me2vu6v8lhssdkrpenp1uunl9s3bdc8njqp"
+	staging   = "mcf9oimts3laqli01e2ktrjdudkdbh8dg42a"
+	accountID = configAPI.GetActiveProfileAccountID()
+)
+
+type Service struct {
+	client *client.SplitClient
+}
+
+// Creates a new instance of a Split Factory
+// Using "localhost" as the apiKey allows us to use Split.io
+// in localhost mode as defined in their documentation
+func NewSplitService(apiKey string, region string) (*Service, error) {
```

Comment:
nit: rename to `NewService`

Reply:
How will users be able to pass API keys if they are not exported from this package?  Also consider if it's worth abstracting the concept of keys from the user and just providing a simple prod/staging switch (bool or enum).  This could allow for initialization with only a single argument rather than the two here.